shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -1092,4 +1094,15 @@ - (void)testMouseSupport API_AVAILABLE(ios(13.4)) {
       dispatchPointerDataPacket:std::make_unique<flutter::PointerDataPacket>(0)];
 }
 
+- (void)testFakeEventTimeStamp {
+  FlutterViewController* vc = [[FlutterViewController alloc] initWithEngine:self.mockEngine
+                                                                    nibName:nil
+                                                                     bundle:nil];
+  XCTAssertNotNil(vc);
+
+  flutter::PointerData pointer_data = [vc generatePointerDataForFake];
+  int64_t current_time = [[NSProcessInfo processInfo] systemUptime];
+  XCTAssertTrue(current_time / 2 == pointer_data.time_stamp / 1000 / 1000 / 2,
```

Comment:
Why the `/ 2`?

Reply:
Try to avoid testing boundary, such as pointer_data.time_stamp is equal to 1999999 microseconds, current_time 2000000 microseconds.  `/2` is a wrong way. I have fixed it.